src/core/include/openvino/op/roi_align_rotated.hpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/op/util/roi_align_base.hpp"
+
+namespace ov {
+namespace op {
+namespace v14 {
+/// \brief ROIAlignRotated operation.
+///
+/// \ingroup ov_ops_cpp_api
+class OPENVINO_API ROIAlignRotated : public util::ROIAlignBase {
+public:
+    OPENVINO_OP("ROIAlignRotated", "opset14", util::ROIAlignBase);
+
+    ROIAlignRotated() = default;
+    /// \brief Constructs a ROIAlignRotated operation.
+    ///
+    /// \param clockwise_mode  If true, rotation angle is interpreted as clockwise, otherwise as counterclockwise
+    ROIAlignRotated(const Output<Node>& input,
```

Comment:
Describe all parameters.

Reply:
Those parameter are already described in a base. Will add a comment about that fact, since copying it is a wrong idea.

Reply:
These comments are used for docs generation and in IDE helper pop-ups. The other, more important, reason to have it fully described is implicit inconsistency once the base class changes.
Please apply description of all parameters to both versions.

Reply:
> The other, more important, reason to have it fully described is implicit inconsistency once the base class changes

Exactly - what you suggest here might introduce inconsistency, because you will have the same param described in a few places. Somebody might change the def of some param and forget to update the comments in all the paces. This simply breaks a single definition rule. Because of that, every param should be described only once and the base class it is where it should be done, since it is where those params are defined.

Having said that, if the problem is that the tool used for generation docs cannot handle it i might to do as you suggest as an exception, but that tool should be fixed as soon as possible. Otherwise I disagree with this comment.